### PR TITLE
Add 'Fork me on Github' ribbon to landing page - story# 204

### PR DESCRIPTION
**Pull request by tnewell5**
Hey Travis!

I chose gray for the ribbon color. Let me know if you prefer another color.

Thanks!
Tanya

### CODE REVIEW

**Comment by vanderhoop** [comment]
hey @tnewell-nbc this adds the fork me on GitHub to every page. it should only be visible on the landing page. can you make that change and ping me? Thanks!

**Comment by tnewell5** [comment]
hey @vanderhoop I've made the changes. Let me know if it's good to go!
Thanks,
Tanya

File: web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -12,6 +12,9 @@
   </head>
 
   <body class="<%= if @view_module.include_backdrop, do: "photo-backdrop" %>">
+    <%= if @view_module.include_backdrop do %>
```
Comment (vanderhoop) [inline web/templates/layout/app.html.eex]: hey @tnewell5, there's a specific template for the landing page. This is the app.layout.template that _wraps_ all other templates. Put the ribbon in the non-layout template so we can avoid the conditional!

**Comment by tnewell5** [comment]
hey Travis! 
How about now?

Thanks!
Tanya

**Comment by vanderhoop** [comment]
oui oui oui!

**Pull request merged**